src/FESpaces/DivConformingFESpaces.jl

```diff
@@ -28,8 +28,11 @@ struct TransformRTDofBasis{Dc,Dp} <: Map end ;
 
 function get_cell_dof_basis(model::DiscreteModel,
                             cell_reffe::AbstractArray{<:GenericRefFE{RaviartThomas}},
-                            ::DivConformity)
-    sign_flip = get_sign_flip(model, cell_reffe)
+                            ::DivConformity,
+                            sign_flip=nothing)
+    if (sign_flip==nothing)
```

Comment:
Why not simply use `sign_flip = get_sign_flip(model, cell_reffe)`in the optional argument?

Reply:
Done.